storage/cloud.py

```diff
@@ -1009,12 +1010,12 @@ def __init__(
         self.cloudfront_privatekey = self._load_private_key(cloudfront_privatekey_filename)
 
     def get_direct_download_url(
-        self, path, request_ip=None, expires_in=60, requires_cors=False, head=False
+        self, path, request_ip=None, expires_in=60, requires_cors=False, head=False, user=None
```

Comment:
nit: Unless we need the full context, I would just pass the username as a `str`, or `**kwargs`, so that in the future we more parameters are needed, they can just be passed arbitrarily.